### PR TITLE
Add script to push jar file to nexus

### PR DESCRIPTION
**Pull request by tonyastolfi**
This change is [Reviewable](https://reviewable.io/reviews/datatamer/elasticsearch/5)

### CODE REVIEW

**Comment by laferrieren** [comment]
Some docs would be nice, a comment at the head of the file or on the readme would be nice

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/datatamer/elasticsearch/5#-:-KeJp0qpxXlFn--08Eya:bufuq3p)*

File: pushToTamrNexus.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+function fullfile {
+    echo $(cd $(dirname $1) && pwd)/$(basename $1)
+}
+
+function pushjar {
+    if [ $# -lt 4 ]; then
+        echo "usage: $0 <file> <groupId> <artifactId> <version>"
+        return 1
+    fi
+    mvn deploy:deploy-file -Dfile=$(fullfile $1) \
```
Comment (data-tamer-vgp) [inline pushToTamrNexus.sh:13]: Might want to check for Maven version being some minimum. Older versions won't do the right thing.

**Review by data-tamer-vgp — CHANGES_REQUESTED** [review]

**Comment by tonyastolfi** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[pushToTamrNexus.sh, line 13 at r1](https://reviewable.io:443/reviews/datatamer/elasticsearch/5#-KeGywI0En8w7ydz_Zij-r1-13:-KedC_byKpYAUa-I9GnY:b-896fix) ([raw file](https://github.com/datatamer/elasticsearch/blob/249b9549a9dc4673c24aa3a857ba3cd71821fb80/pushToTamrNexus.sh#L13)):*
<details><summary><i>Previously, data-tamer-vgp wrote…</i></summary><blockquote>

Might want to check for Maven version being some minimum. Older versions won't do the right thing.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/datatamer/elasticsearch/5)*

**Review by data-tamer-vgp — APPROVED** [review]

**Comment by data-tamer-vgp** [comment]
Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/datatamer/elasticsearch/5)*

**Comment by laferrieren** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/datatamer/elasticsearch/5#-:-KeddXSz_zH8gYi91vXt:bnfp4nl)*

**Pull request merged**